src/components/Search/Search.component.jsx

```diff
@@ -1,16 +1,46 @@
-import React from 'react';
+import React, { useRef } from 'react';
+import { useHistory } from 'react-router-dom';
 
 import classes from './Search.module.css';
+import { useGetVideos } from '../../utils/hooks/useGetVideos';
+
+const Search = ({ onClose, isOpen }) => {
+  const searchInputRef = useRef();
+
+  const { getVideos } = useGetVideos();
+  const history = useHistory();
+
+  const getData = (evt) => {
+    evt.preventDefault();
+
+    const searchInputValue = searchInputRef.current.value;
```

Comment:
everything related to the input value is better if you create a state for it. An on change on the input would hep to handle this and then you only use the value.